src/api/app/views/layouts/webui/_places.html.haml

```diff
@@ -8,10 +8,16 @@
         = link_to(user_path(User.session!), id: 'link-to-user-home', class: 'nav-link') do
           = image_tag_for(User.session, size: 23, custom_class: 'rounded-circle bg-light mr-2')
           Profile
-    %li.nav-item
-      = link_to(my_tasks_path, class: 'nav-link', title: 'Tasks') do
-        %i.fas.fa-tasks.fa-lg.mr-2
-        %span.nav-item-name Tasks
+    -# Without this, we would have two links to the tasks, one here and one in the top navigation.
+       This is going to be resolved once the notifications_redesign is rolled out.
```

Comment:
```suggestion
    -# This is going to be resolved once the notifications_redesign is rolled out.
```

Reply:
An alternative for multiline comments is:

```
/
  line 1
  line 2
```

or 

```
/ -------------------------------------------
/ Comments
/ -------------------------------------------
```

According to this: https://learnxinyminutes.com/docs/haml/

Reply:
Both pass linters for me.

Reply:
Sure, but this is already working, so I don't see why it's needed to change. The text `This is going to be resolved once the notifications_redesign is rolled out.` isn't displayed in the view, so as expected, it's processed as a comment.

Reply:
This is just a comment, not a change request. Feel free to add it or not.

Reply:
If I had to choose, I would use:

```
/ -------------------------------------------
/ Comments
/ -------------------------------------------
```